Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed.py

```diff
@@ -0,0 +1,413 @@
+
+
+class SetList(list):
```

Comment:
looks like overkill.
just create a dict of the form: `{ x['value']: x}`
* solves duplication problem: check if the key exists
* can be converted to list using `.values()` and then sorted as you wish